packages/remix-adapter/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@netlify/remix-adapter",
+  "version": "1.0.0",
+  "description": "Remix Adapter for Netlify Functions",
+  "types": "./dist/index.d.ts",
+  "main": "./dist/index.js",
+  "exports": "./dist/index.js",
+  "files": [
+    "dist/",
+    "CHANGELOG.md",
+    "LICENSE.md",
+    "README.md"
+  ],
+  "scripts": {
+    "prepack": "npm run clean && npm run build",
+    "build": "tsc --build tsconfig.json",
+    "build:watch": "npm run build -- --watch",
+    "clean": "rm -rf build"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/netlify/remix-compute",
+    "directory": "packages/remix-adapter"
+  },
+  "keywords": [
+    "remix",
+    "netlify"
+  ],
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/netlify/remix-compute/issues"
+  },
+  "homepage": "https://github.com/netlify/remix-compute#readme",
+  "dependencies": {
+    "@remix-run/node": "1.16.0"
```

Comment:
```suggestion
    "@remix-run/node": "1.17.0"
```

Reply:
_probably_ better to do `"@remix-run/node": "^1.17.0"`

my local npm config at the time had exact pinned deps configured

Reply:
I've bumped it to latest, 1.18.1